servicehandler/portsscan.go

```diff
@@ -0,0 +1,50 @@
+package servicehandler
+
+import (
+	"context"
+
+	logger "github.com/kubescape/go-logger"
+	"github.com/kubescape/kubescape-network-scanner/cmd"
+	v1 "k8s.io/api/core/v1"
+)
+
+type Port struct {
+	port              int
+	protocol          string
```

Comment:
would be nicer to have it as defined type with set of values, or iota

Reply:
can you explain further please?